www/front_src/src/Navigation/Sidebar/Menu/index.tsx

```diff
@@ -0,0 +1,204 @@
+
+import { equals, isNil } from 'ramda';
+import { useNavigate } from 'react-router-dom';
+import { useAtom } from 'jotai';
+
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import makeStyles from '@mui/styles/makeStyles';
+
+import { useMemoComponent } from '@centreon/ui';
+
+import { Page } from '../../models';
+import {
+  navigationItemSelectedAtom,
+  propsNavigationItemSelected,
+} from '../sideBarAtoms';
+import { closedDrawerWidth, openedDrawerWidth } from '../index';
+
+import MenuItems from './MenuItems';
+import icons from './icons';
+import CollapsableItems from './CollapsableItems';
+
+interface Props {
+  isDrawerOpen: boolean;
+  navigationData?: Array<Page>;
+}
+
+const useStyles = makeStyles((theme) => ({
+  icon: {
+    color: theme.palette.text.primary,
+    fontSize: 26,
+  },
+  list: {
+    '&.MuiList-root': {
+      padding: theme.spacing(0, 0, 0, 0),
+    },
+  },
+}));
+
+const NavigationMenu = ({
+  isDrawerOpen,
+  navigationData,
+}: Props): JSX.Element => {
+  const classes = useStyles();
+  const navigate = useNavigate();
+
+  const [hoveredIndex, setHoveredIndex] = useState<number | null>(null);
+  const [currentTop, setCurrentTop] = useState<number>();
+  const [collapseScrollMaxHeight, setCollapseScrollMaxHeight] = useState<
+    number | undefined
+  >(undefined);
+  const [collapseScrollMaxWidth, setCollapseScrollMaxWidth] = useState<
+    number | undefined
+  >(undefined);
+  const [navigationItemSelected, setNavigationItemSelected] = useAtom(
+    navigationItemSelectedAtom,
+  );
+  const levelName = 'level_0_Navigated';
+  const currentWidth = isDrawerOpen ? openedDrawerWidth / 8 : closedDrawerWidth;
+
+  const props = {
+    collapseScrollMaxHeight,
+    collapseScrollMaxWidth,
+    currentTop,
+    currentWidth,
+    hoveredIndex,
+    isDrawerOpen,
+    level: 1,
+    setCollapseScrollMaxHeight,
+    setCollapseScrollMaxWidth,
+  };
+
+  const hoverItem = (
+    e: React.MouseEvent<HTMLElement>,
+    index: number | null,
+    item: Page,
+  ): void => {
+    const rect = e.currentTarget.getBoundingClientRect();
+    const { top } = rect;
+    setCurrentTop(top - 0.58);
+    setHoveredIndex(index);
+    setNavigationItemSelected({
+      ...navigationItemSelected,
+      level_0: { index, label: item.label, url: item?.url },
+    });
+  };
+
+  const handleLeave = (): void => {
+    setHoveredIndex(null);
+  };
+
+  const getUrlFromEntry = (entryProps: Page): string | null | undefined => {
+    const page = !isNil(entryProps?.page) ? entryProps.page : '';
+    const options = !isNil(entryProps?.options) ? entryProps.options : '';
+
+    const urlOptions = `${page}${options}`;
+    const url = entryProps.is_react
+      ? entryProps.url
+      : `/main.php?p=${urlOptions}`;
+
+    return url;
+  };
+
+  const addNavigationItemSelected = (nivel: number): void => {
+    if (navigationItemSelected) {
+      Object.keys(navigationItemSelected).forEach((i: string) => {
```

Comment:
navigationItemSelected cannot be modified direcly because it's an atom, that's why we should not mutate it but set a new object with the updated properties